fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -3349,6 +3349,33 @@ ACTOR Future<Void> processTransactionStateRequestPart(TransactionStateResolveCon
 
 } // anonymous namespace
 
+// Logs metrics related to workload burstiness on a much more frequent interval
+// than the typical five second ProxyMetrics trace.
+ACTOR Future<Void> logDetailedMetrics(ProxyCommitData const* commitData) {
+	state int64_t mutationsBaseline = 0;
+	state int64_t mutationBytesBaseline = 0;
+
+	loop {
+		wait(delay(SERVER_KNOBS->BURSTINESS_METRICS_LOG_INTERVAL));
+
+		int64_t mutationsReal = commitData->stats.mutations.getValue();
+		int64_t mutationBytesReal = commitData->stats.mutationBytes.getValue();
+
+		TraceEvent("ProxyDetailedMetrics")
+		    .detail("Interval", SERVER_KNOBS->BURSTINESS_METRICS_LOG_INTERVAL)
```

Comment:
This should be "Elapsed", and the actual elapsed time since we reset the counters.

Reply:
This is probably not needed, since each event already have `Time` and `DateTime` and it's fairly easy to derive the interval.

Reply:
We do log elapsed for `CounterCollections` in general though, so it might be good to match that behavior.